raspberryPI3/ros2_ws/src/imu_tools/imu_filter_madgwick/src/imu_filter_ros.cpp

```diff
@@ -448,6 +448,7 @@ void ImuFilterMadgwickRos::publishFilteredMsg(
     // create and publish filtered IMU message
     ImuMsg imu_msg = *imu_msg_raw;
 
+    imu_msg.header.frame_id = FRAME_ID;
```

Comment:
Already defined line 467: `    imu_msg.header.frame_id= imu_frame_;`
is it normal?

Reply:
Definitely no haha, I removed it